src/vue/pages/Operations.vue

```diff
@@ -0,0 +1,244 @@
+        <span class="op-history__filters-text">
+          {{ 'op-pages.show' | globalize }}:
+        </span>
+        <select-field
+          v-if="isLoaded"
+          v-model="tokenCode"
+          :values="tokens"
+          class="app__no-border-select"
+        />
+      </div>
+      <div slot="extra">
+        <button
+          v-ripple
+          class="app__button-raised"
+          @click="isWithdrawalDrawerShown = true"
+        >
+          <i class="mdi mdi-download op-history__btn-icon" />
+          {{ 'op-pages.withdrawal' | globalize }}
+        </button>
+      </div>
+    </top-bar>
+    <drawer :is-shown.sync="isWithdrawalDrawerShown">
+      <template slot="heading">
+        {{ 'withdrawal-form.withdrawal' | globalize }}
+      </template>
+      <withdrawal-form @cancel="isWithdrawalDrawerShown = false" />
+    </drawer>
+    <template v-if="!isLoadFailed">
+      <div class="op-history__list" v-if="isLoaded">
+        <template v-if="operations.length">
+          <op-list :list="operations" />
+        </template>
+        <template v-else>
+          <div class="op-history__no-transactions">
+            <i class="op-history__no-tx-icon mdi mdi-trending-up" />
+            <h2>{{ 'op-pages.no-operation-history' | globalize }}</h2>
+            <p>{{ 'op-pages.here-will-be-the-list' | globalize }}</p>
+          </div>
+        </template>
+        <collection-loader
+          :first-page-loader="pageLoader"
+          @first-page-load="setFirstPageData"
+          @next-page-load="setNextPageData"
+        />
+      </div>
+    </template>
+    <template v-else>
+      <div class="op-history__error">
+        <i class="op-history__error-icon mdi mdi-comment-alert-outline" />
+        <h2>{{ 'op-pages.something-went-wrong' | globalize }}</h2>
+        <p>{{ 'op-pages.can-not-load-assets' | globalize }}</p>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import SelectField from '@/vue/fields/SelectField'
+import CollectionLoader from '@/vue/common/CollectionLoader'
+import TopBar from '@/vue/common/TopBar'
+import Drawer from '@/vue/common/Drawer'
+import WithdrawalForm from '@/vue/forms/WithdrawalForm'
+import OpList from '@/vue/common/OpList'
+import { Sdk } from '@/sdk'
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex/types'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import { RecordWrapper } from '@/js/records'
+import { MatchRecord } from '@/js/records/operations/match.record'
+
+export default {
+  name: 'history-index',
+  components: {
+    SelectField,
+    CollectionLoader,
+    TopBar,
+    Drawer,
+    WithdrawalForm,
+    OpList,
+  },
+  data: _ => ({
+    tokenCode: null,
+    assets: [],
+    operations: [],
+    isLoaded: false,
+    isLoadFailed: false,
+    isWithdrawalDrawerShown: false,
+    pageLoader: () => {},
+  }),
+  computed: {
+    ...mapGetters([
+      vuexTypes.account,
+      vuexTypes.accountId,
+    ]),
+    tokens () {
```

Comment:
don't use the word `token` at all. it means the same as `asset`, and having both terms is very confusing

Reply:
`assetCodes` will be amazing here

Reply:
ok